source/proof-of-concept-guide/aws-infrastructure-monitoring.rst

```diff
@@ -80,7 +78,7 @@ You can also find additional :ref:`CloudTrail use cases <cloudtrail-use-cases>`
 Visualize the alerts
 --------------------
 
-You can visualize the alert data in the Wazuh dashboard. To do this, navigate through **Modules** > **Amazon AWS** module.
+You can visualize the alert data in the Wazuh dashboard. To do this, navigate through **AWS** application.
```

Comment:
**nitpick**: The application is called `Amazon Web Services`. Change from `AWS` to `Amazon Werb Services`.